x-pack/plugins/alerting/server/lib/create_alert_event_log_record_object.ts

```diff
@@ -52,13 +57,23 @@ export function createAlertEventLogRecordObject(params: CreateAlertEventLogRecor
     consumer,
     spaceId,
     flapping,
+    alertSummary,
   } = params;
   const alerting =
-    params.instanceId || group
+    params.instanceId || group || alertSummary
       ? {
           alerting: {
             ...(params.instanceId ? { instance_id: params.instanceId } : {}),
             ...(group ? { action_group_id: group } : {}),
+            ...(alertSummary
```

Comment:
nit: maybe a unit test for this?

Reply:
Resolved in this commit [4889843](https://github.com/elastic/kibana/pull/150088/commits/4889843abf7ace661c036545ebd50b723872ed5c)